app/models/user.rb

```diff
@@ -351,7 +356,13 @@ def editing_self?(options = {})
       options[:id].to_i == self.id ||
     options[:controller].to_s =~ /\Aapi\/v\d+\/users\Z/ &&
       options[:action] =~ /show|update/ &&
-      (options[:id].to_i == self.id || options[:id] == self.login)
+      (options[:id].to_i == self.id || options[:id] == self.login) ||
```

Comment:
@dLobatog : This method looks like a code smell to me. I don't see how we can get rid of the method, though. Any suggestions?